packages/react-scripts/scripts/utils/createJestConfig.js

```diff
@@ -39,7 +39,6 @@ module.exports = (resolve, rootDir, isEjecting) => {
       '<rootDir>/src/**/*.{spec,test}.{js,jsx,ts,tsx}',
     ],
     testEnvironment: 'jsdom',
-    testRunner: require.resolve('jest-circus/runner'),
```

Comment:
Note to self:

```
Note: As of Jest 27, jest-circus is the default test runner, so you do not have to install it to use it.
```

Reply:
yes @petetnt , right so that is why we dont need to specify a testRunner  , if its  jest-circus already by default

Reply:
Good catch, thanks!